src/ngraph/pass/manager.cpp

```diff
@@ -37,12 +37,12 @@ using namespace ngraph;
 
 pass::Manager::Manager()
 {
-    static const auto nevt = std::getenv("NGRAPH_ENABLE_VISUALIZE_TRACING");
+    static const auto nevt = std::getenv("NGRAPH_ENABLE_DEBUG_VISUALIZE_TRACING");
```

Comment:
why change the name here? Can we also follow a deprecation path here since they are bridges/scripts using this and docs that will need to be changed.

Reply:
reverted